tasks/InstallBundletool/install-bundletool.ts

```diff
@@ -63,9 +68,21 @@ async function run() {
     }
 }
 
-async function downloading(fileName: string, url: string) {
+async function downloading(fileName: string, url: string, downloadedFile:string) {
```

Comment:
Their is something strange here, the fileName property is not used, you probably broke something.

Reply:
I also updated some files.

Reply:
I will take a look 😅